client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -2,34 +2,32 @@ import { Button, Gridicon } from '@automattic/components';
 import classnames from 'classnames';
 import { translate, useRtl } from 'i18n-calypso';
 import Badge from 'calypso/components/badge';
-import { isTaskDisabled, hasIncompleteDependencies } from './task-helper';
 import { Task } from './types';
 
 const ChecklistItem = ( { task }: { task: Task } ) => {
 	const isRtl = useRtl();
-	const { id, isCompleted, keepActive, title, actionDispatch } = task;
-	const taskDisabled = isTaskDisabled( task );
+	const { id, completed, disabled, title, actionDispatch } = task;
 
 	// Display chevron if task is incomplete. Don't display chevron and badge at the same time.
-	const shouldDisplayChevron =
-		! hasIncompleteDependencies( task ) && ! isCompleted && ! task.badgeText;
+	const shouldDisplayChevron = ! completed && ! disabled && ! task.badgeText;
 
 	return (
 		<li
 			className={ classnames( 'launchpad__task', {
-				'completed-and-active': isCompleted && keepActive, // a task that is completed and can be revisited
-				'completed-and-inactive': isCompleted && ! keepActive, // a task that is completed and can't be revisited
-				'not-completed': ! isCompleted && ! keepActive, // a task that hasn't been completed yet
+				completed: completed,
+				pending: ! completed,
```

Comment:
The `pending` and `enabled` css classes are just "syntactic sugar" for easier styling. Alternatively we had to use `:not()` selectors which increase the complexity and lower the readability